scripts/list-packages

```diff
@@ -7,6 +7,8 @@ const child_process = require('child_process');
 const fs = require('fs-extra');
 const path = require('path');
 
+const monolithicPackages = ['monocdk', 'aws-cdk-lib'];
```

Comment:
Since we create the list of packages to pack using `lerna ls --toposort --json` which ignores private packages, it will not include `aws-cdk-lib`, so this is somewhat redundant.future proofing.